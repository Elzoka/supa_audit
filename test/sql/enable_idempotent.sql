begin;

    create table public.dummy(
        id int primary key
    );


    select audit.enable_tracking('public.dummy');
    select audit.enable_tracking('public.dummy');


    select tgname
    from pg_trigger
    where tgrelid = 'public.dummy'::regclass;

rollback;
