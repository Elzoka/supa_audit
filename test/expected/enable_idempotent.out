begin;
    create table public.dummy(
        id int primary key
    );
    select audit.enable_tracking('public.dummy');
 enable_tracking 
-----------------
 
(1 row)

    select audit.enable_tracking('public.dummy');
 enable_tracking 
-----------------
 
(1 row)

    select tgname
    from pg_trigger
    where tgrelid = 'public.dummy'::regclass;
   tgname    
-------------
 audit_i_u_d
 audit_t
(2 rows)

rollback;
